src/pages/contacts.vue

```diff
@@ -196,6 +196,7 @@ export default Vue.extend({
       if (!val) {
         this.inputtedName = "";
         this.inputtedWallet = "";
+        this.modalError = "";
```

Comment:
@JackHamer09 move it to the computed property. Avoid using data for the errors processing

Reply:
In this case we want to show error only after user clicks the button. If we just use computed property we would always have some error when user opens the modal and there is a possibility it would be jumping (error appearing and isappearing) while user is typing. Another options is to create custom inputs components that would each have their own errors and other stuff (like it's done with address component), but I think its an overkill for now and it's just easier to leave it like that untill we switch to a UI library.